config/initializers/job_configurations.rb

```diff
@@ -3,6 +3,8 @@
 
 cron_5m = '0/5 * * * *'
 interval_5m = 5 * 60
+cron_1h = '0/ * * * *'
```

Comment:
Is the slash going to cause trouble?

```suggestion
cron_1h = '0 * * * *'
```

Reply:
it's probably not going to help... today is definitely a Monday... thanks for the catch